backend/pyproject.toml

```diff
@@ -9,6 +9,8 @@ python = "^3.10"
 Django = "^3.2"
 django-environ = "^0.9.0"
 psycopg2 = "^2.9.3"
+drf-oidc-auth = "^3.0.0"
+social-auth-app-django = "5.0.0"
```

Comment:
`social-auth-app-django` version 5.1.0 has some [migration issues](https://github.com/python-social-auth/social-app-django/issues/429) that break our CI, so at the moment this has to be pinned to 5.0.0.

Reply:
This has been solved with 5.2.0